i6_models/parts/blstm.py

```diff
@@ -0,0 +1,84 @@
+from dataclasses import dataclass
+import torch
+from torch import nn
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass()
+class BlstmEncoderV1Config(ModelConfiguration):
+    """
+    Attributes:
+        num_layers: number of bi-directional LSTM layers, minimum 2
+        input_dim: input dimension size
+        hidden_dim: hidden dimension of one direction of LSTM, the total output size is twice of this
+        dropout: nn.LSTM supports internal Dropout applied between each layer of BLSTM (but not on input/output)
+        enforce_sorted:
+            True: expects that sequences are sorted by sequence length in decreasing order.
+                Will not do any sorting.
+                This is required for ONNX-Export.
+            False: no expectation.
+                It will internally enforce that they are sorted
+                and undo the reordering at the output.
+
+            Sorting can for example be performed independent of the ONNX export in e.g. train_step:
+
+                audio_features_len, indices = torch.sort(audio_features_len, descending=True)
+                audio_features = audio_features[indices, :, :]
+                labels = labels[indices, :]
+                labels_len = labels_len[indices]
+    """
+
+    num_layers: int
+    input_dim: int
+    hidden_dim: int
+    dropout: float
+    enforce_sorted: bool = True
```

Comment:
I am not sure about this. The parameter internally is named enforce_sorted. This might cause confusion.

Reply:
While the name is certainly not optimal, I would still stick to it because of what Benedikt says, but also because the RuntimeError you get if you do it wrong clearly mentions `enforce_sorted` as parameter name.

Reply:
Okay, that's a good reason, then I'm fine with the name.